src/net/sourceforge/kolmafia/session/ChoiceManager.java

```diff
@@ -2146,6 +2146,7 @@ public static void postChoice0(final String urlString, final GenericRequest requ
   public static boolean bogusChoice(final GenericRequest request) {
     if (!ChoiceManager.handlingChoice
         || !request.getURLString().startsWith("choice.php")
+        || "choice.php".equals(request.getURLString())
```

Comment:
This will leave ChoiceManager.handlingChoice as true.
Sometimes that is correct.
But in the Whoops case, it is not correct; we'd like to exit choice processing immediately.

Reply:
If you copy the newly added test to your PR I am happy to close this one

Reply:
OK. I'm adding two tests:

Bogus message and raw choice.php: returns true but no abort
Bogus message and choice.php with arguments: returns true and abort state

Reply:
Perfect 👍